ChangeLog

```diff
@@ -35,37 +39,64 @@
+   * Change: Always initialize SDL before Qt which may solve some crashes on Linux systems (commit:ccb70cb68040976e6823dfc6483d5b89d0496aea, #401)
    * Fix: Truck getting stuck when picking up artifact in tutorial due to a cliff (commit:f4fa0a93e26103402e77e2072e3004b533029904, ticket:4866)
    * Fix: Throttling of production console messages (commit:3bb244b7a41237278a731bde7cc49626646a1107, #327, #329)
-   * Fix: Undefined behavior (commit:41af29c6a58431042cd2649b8aa1984c8e53ef0a, commit:43a34a886c0c62f92065c0c9646bb3027d6b09b6, commit:fa02178faa1caefbbff63d82d9262ad5ba80703f, #351, #353, #356)
+   * Fix: Undefined behavior (commit:41af29c6a58431042cd2649b8aa1984c8e53ef0a, commit:43a34a886c0c62f92065c0c9646bb3027d6b09b6, commit:fa02178faa1caefbbff63d82d9262ad5ba80703f, commit:070f0c8532007cc3f90d1124ed8d76ecf04d22be, #351, #353, #356, #436)
+   * Fix: Tutorial issues (commit:18a0b6ab3f4053948534155b94acc02993e685b0, commit:d79c2a362451af725de5dce7a7cf7677041b9088, commit:f41f78a9cf7606286c4305d49ece5cfda583e1d4, commit:518b2563bf4c77a3391017523e9c401f33144c2c, #381)
+   * Fix: Being able to use saved combat templates before building an HQ (commit:7d6210510ed6efc18694c352b00a66ebea1ccb5c, commit:9e22db301aa93182114d35bf298459ee31eec383, commit:15ec435836aa7fe906f77ed42e555b39789dc67b, #378, #394)
+   * Fix: Debug menu crashing when trying to open it after loading a skirmish save (commit:f340d89f8b8ab3c978dc246a07210438da6e928f, #422)
+   * Fix: Missing AI difficulty button when everyone is on the same team (commit:63c358c85b507c7382abc27785de3148578e41f8, #423)
+   * Fix: Always make sure the default AI stays as the first list entry (commit:6b91d9e5019401eccbe1e23c2ed3d3321f2f0407, #419, ticket:4791)
+   * Fix: "biffer baker" and "double up" cheats causing unintended behavior (commit:6f24a6ceb94ca4e2ec27d926b2c1f9bdfc5fcc4f, #438, ticket:4795)
+   * Fix: Crash when loading a saved game of a challenge from Command line interface (commit:244fd2cbcb12465866191dcc7203aca7719fab8c, #439)
    * Remove: More Qt dependency from the source (commit:97cf508379fe7fb56ac8ecccbe3269783d3620f3, commit:ca9c1166e386f43ff76aa16d6971ca2ab2eb1303, commit:131e4f705888445aaa26643489d6a238a4fd3515, #276, #299)
+ * Modding:
+   * Fix: Desync when trying to use mods while hosting 3rd party maps (commit:7d6210510ed6efc18694c352b00a66ebea1ccb5c, #388)
+   * Fix: Windows hosts crashing when clients connect and mod transfer is attempted (commit:2cddc2fc122ce531314bc7d47c6d279966d4ab04, commit:1b5fe86b6570a2454f189027bcf2fb65aa653db8, #441)
  * Balance:
+   * Add: 3.1 weapon ranges/accuracy with fixes (commit:7f4c03ce5236611ba2c485b5a27f3a445ec47b48, commit:20345ce333248703a1198f84aaad974cf4c86c68, commit:b1d5f8e5885faa4acee9cc01191606818b7d3f03, #387)
+   * Add: 3.1 weapon ranges/accuracy with fixes (commit:c8ecd75d7b426b325202f78ac30f9fa509f27b7e, commit:e6ebad0d3f62ff3d764a4922bb0ea39e82f20bce, #399)
```

Comment:
I suggest changing this description to "Add: Restored campaign weapon
range and accuracy stats of version 3.1, with small changes"